ens/main.py

```diff
@@ -56,7 +57,9 @@
     )
 
 
-ENS_MAINNET_ADDR = ChecksumAddress(HexAddress('0x314159265dD8dbb310642f98f50C066173C1259b'))
+ENS_MAINNET_ADDR = ChecksumAddress(
+    HexAddress(HexStr('0x314159265dD8dbb310642f98f50C066173C1259b'))
```

Comment:
Oh, that's awkward..... that we have to do the double wrap.

Reply:
Yeah, I can't say I'm a huge fan. It'll probably be somewhat confusing/frustrating to devs as it breaks their mypys - at least initially, until they get used to the convention. But, I'm not so sure the awkwardness is worth relaxing the `HexAddress` back to `str`?